pkg/translator/zipkin/zipkinv1/json.go

```diff
@@ -359,7 +359,7 @@ func epochMicrosecondsToTimestamp(msecs int64) pcommon.Timestamp {
 	if msecs <= 0 {
 		return pcommon.Timestamp(0)
 	}
-	return pcommon.Timestamp(uint64(msecs) * 1e6)
+	return pcommon.Timestamp(uint64(msecs) * 1e3)
```

Comment:
It looks like the name of the func is wrong, but the timestamp is in usec, see https://github.com/openzipkin/zipkin-api/blob/master/zipkin-api.yaml#L385

Reply:
Talked too soon, you are correct we convert to nanos, so `1e3` is correct.